config.yaml

```diff
@@ -41,6 +41,14 @@ options:
     type: string
     description: "Hostname that email sent by this discourse should appear to come from."
     default: ""
+  smtp_enable_start_tls:
+    type: string
+    description: "enable TLS encryption for smtp connections"
```

Comment:
Other descriptions start with a capital letter and end with a dot.

Reply:
Fixed